Source/WebCore/platform/network/cocoa/CookieCocoa.mm

```diff
@@ -124,10 +124,7 @@ static NSHTTPCookieStringPolicy _Nullable nsSameSitePolicy(Cookie::SameSitePolic
     , commentURL { cookie.commentURL }
     , ports { portVectorFromList(cookie.portList) }
 {
-ALLOW_NEW_API_WITHOUT_GUARDS_BEGIN
-    if ([cookie respondsToSelector:@selector(sameSitePolicy)])
-        sameSite = coreSameSitePolicy(cookie.sameSitePolicy);
-ALLOW_NEW_API_WITHOUT_GUARDS_END
+    sameSite = coreSameSitePolicy(cookie.sameSitePolicy);
```

Comment:
Added in macOS 10.15 ✅ 

https://developer.apple.com/documentation/foundation/nshttpcookie/3042420-samesitepolicy